src/schema/core.yaml

```diff
@@ -1006,6 +1006,9 @@ classes:
       - lbceq
     slot_usage:
 
+      temp:
+        range: QuantityValueCelsius
```

Comment:
Should this range be on the global definition of this slot or is there a reason it's on the `slot_usage` for biosample only?